peers/peerpool.go

```diff
@@ -201,12 +201,12 @@ func (p *PeerPool) stopDiscovery(server *p2p.Server) {
 		return
 	}
 
-	for _, t := range p.topics {
-		t.StopSearch(server)
-	}
 	if err := p.discovery.Stop(); err != nil {
 		log.Error("discovery errored when was closed", "err", err)
```

Comment:
Unrelated, but can we rephrase this as `discovery errored when stopping` to make it clearer?